src/Entity/Capture.php

```diff
@@ -145,8 +145,14 @@ public function setEventSource($eventSource)
     public function setEventDate(DateTime $eventDate)
     {
         // Lift Profile Manager is currently accepting this ISO 8601 format:
-        // "2017-02-07T19:59:53.123456Z".
-        $this['event_date'] = $eventDate->format('Y-m-d\TH:i:s.u\Z');
+        // "2017-02-07T19:59:53.123Z" (or "2017-02-07T19:59:53.000Z" pre 7.0 ).
+        // todo: the above comment needs updating
+        // todo: works for all but hhvm-3.18.5
+        $format = 'Y-m-d\TH:i:s.v\Z';
+        if (version_compare(phpversion(), '7.0', '<')) {
+            $format = 'Y-m-d\TH:i:s.000\Z';
```

Comment:
@matthewmorley What about use something like `->format(DateTime::ISO8601)`?

Reply:
We should discuss with the team(s), I dont know what the actual requirement/desire is around fraction of a second granularity. Sometimes it is useful, even if not a requirement... 

Using DateTime::ISO8601 looks like: https://3v4l.org/MoEpT which will certainly be more predictable.

Reply:
@matthewmorley My team has no requirements for microsecond granularity. Just seconds would be OK.